.github/workflows/hrm-ecs-export.yml

```diff
@@ -0,0 +1,165 @@
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ inputs.region }}
+          # this plugin sets the AWS account ID to a secret which is not allowed in outputs
+          # we have to disable that so env.DOCKER_IMAGE will work
+          mask-aws-account-id: 'no'
+      
+      # Run the task definition and add to env
+      - name: Run Task Definition
+        run: |
+          task_definition=$(aws ecs describe-task-definition --task-definition ${{ env.ECR_REPOSITORY }}-task --query taskDefinition.taskDefinitionArn)
+          echo task_definition=$task_definition >> $GITHUB_ENV
+          echo subnets=$task_definition | jq -r '.networkMode == "awsvpc" | if . then .networkConfiguration.awsvpcConfiguration.subnets | join(",") else empty end' >> $GITHUB_ENV
+          echo = security_group=$task_definition | jq -r '.networkMode == "awsvpc" | if . then .networkConfiguration.awsvpcConfiguration.securityGroups | join(",") else empty end' >> $GITHUB_ENV
+      
+      # Command to trigger the execution of the ECS task. 
+      - name: Trigger ECS Task
+        id: ecs-task
+        run: |
+          aws ecs run-task \
+            --cluster ${{ env.ECR_REPOSITORY }}-task \
+            --launch-type FARGATE \
+            --task-definition ${{ env.task_definition }} \
+            --network-configuration "awsvpcConfiguration={subnets=['env.subnets'],securityGroups=['env.security_group'],assignPublicIp='ENABLED'}" \
+            --overrides '{
+              "containerOverrides": [
+                {
+                  "name": "${{ env.ECR_REPOSITORY }}-scheduled-task",
+                  "command": [
+                    "npm", 
+                    "run", 
+                    "start:khp-data-pull",
+                    "${{ github.event.inputs.start-date }}",
+                    "${{ github.event.inputs.end-date }}"
+                  ]
+                }
+              ]
+            }'
+
+      # Ensures that the workflow doesn't proceed until the ECS task has finished 
+      # executing. This step prevents any potential race conditions or incorrect 
+      # assumptions about the task's status.
+      - name: Wait for ECS Task Completion
+        id: ecs-task-status
+        run: |
+          aws ecs wait tasks-stopped \
+            --cluster ${{ env.ECR_REPOSITORY }}-task \
+            --tasks ${{ steps.ecs-task.outputs.task-id }}
+
+      #  Retrieve the output of the ECS task that was triggered and extract 
+      # the desired information from it. 
+      - name: Get ECS Task Output
+        id: ecs-task-output
+        run: |
+          aws ecs describe-tasks \
+            --cluster ${{ env.ECR_REPOSITORY }}-task \
+            --tasks ${{ steps.ecs-task.outputs.task-id }} \
+            --query "tasks[0].containers[0].command[1:3]" \
+            --output text
```

Comment:
@murilovmachado - There is a task named `Wait for ECS Task Completion`. That task checks to make sure the ECS task is finished executing. We may not be needing the above `Get ECS Task Output`, so I will go ahead and remove it. I added it in case we need to extract data from the ECS task output.